src/components/ModalFlows/TransactionStepper.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { noop, GU } from '@aragon/ui'
+import Stepper from '../Stepper/Stepper'
+import { useWallet } from '../../providers/Wallet'
+
+const indexNumber = {
+  0: 'First',
+  1: 'Second',
+  2: 'Third',
+  3: 'Fourth',
+  4: 'Fifth',
+}
```

Comment:
This is temporary, ideally we will provide step titles via connect data in the future.